packages/workers/src/scripts/mp-liquidity.ts

```diff
@@ -38,16 +38,23 @@ async function getTransactionData(transactionType: string, hash: string): Promis
   const addPath ='https://api.ethplorer.io/getAddressTransactions/';
   const fullPath = `${addPath}${hash}?apiKey=${
       key
-  }&limit=200&timestamp=${
-    oneHoursBefore
-  }`
+  }&limit=200`
 
   const res = await fetch(fullPath);
+
+  console.log('*************');
+  console.log(oneHoursBefore);
+  console.log('*************');
+  console.log(res);
```

Comment:
I don't think we want to log the full response, because that contains headers and a lot of other noise. Can we tighten up what we are logging for?